apps/events-helsinki/src/domain/search/eventSearch/filterSummary/DateFilter.tsx

```diff
@@ -1,9 +1,6 @@
-import {
-  translateValue,
-  FilterButton,
-  useConfig,
-} from 'events-helsinki-components';
+import { translateValue, FilterButton } from 'events-helsinki-components';
 import type { FilterType } from 'events-helsinki-components';
+import { useTranslation } from 'next-i18next';
 import React from 'react';
 
 export interface DateFilterProps {
```

Comment:
The DateFilter could be moved to the common components-package with the SharedLinks